concordium-node/src/bin/cli.rs

```diff
@@ -152,9 +153,33 @@ async fn main() -> anyhow::Result<()> {
         None
     };
 
-    if let Some(ref import_path) = conf.cli.baker.import_path {
+    // Out-of-band catch-up
+    if let Some(ref import_blocks_from) = conf.cli.baker.import_blocks_from {
+        let import_path =
+            data_dir_path.to_path_buf().join(concordium_node::configuration::CATCHUP_FILE);
```

Comment:
Please remove this default name. It just adds "magical" behaviour to the node for no good reason.

I'd recommend using the file name from the URL using https://docs.rs/url/latest/url/struct.Url.html#method.path_segments and only if it is empty or unusable try a default (which would ideally have a temporal component for clarity), making sure to log extensively that this is what you are doing.